CONTRIBUTING.md

```diff
@@ -1,42 +1,68 @@
-# Contributing
+# Contributing to HedgeDoc
 
-When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change.
+Thanks for your help in improving the HedgeDoc project!
 
-Please note we have a [code of conduct](CODE-OF-CONDUCT.md), please follow it in all your
+Please note we have a [code of conduct][code-of-conduct], please follow it in all your
```

Comment:
maybe we should use _comply with_

Reply:
I really don't like "comply" but now use "respect" instead.